ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -257,7 +257,7 @@ static int HandleVnodeOperation(
 
     ProfileSample functionSample(Probe_VnodeOp);
     
-    vfs_context_t context = reinterpret_cast<vfs_context_t>(arg0);
+    vfs_context_t _Nonnull context = reinterpret_cast<vfs_context_t>(arg0);
```

Comment:
I am a bit surprised the compiler does not complain that `arg0` is not flagged as `_Nonnull`.  Do you know if it's because we're casting?